.github/workflows/release.yml

```diff
@@ -0,0 +1,419 @@
+          Write-Host "✅ Renamed installer to: $newName"
+        } else {
+          Write-Host "❌ No x64 installer file found"
+          exit 1
+        }
+
+    - name: Debug - Check installer output
+      run: |
+        echo "=== Checking installer output directory ==="
+        if (Test-Path "Install/Win32/install_image") {
+          echo "✅ install_image directory exists"
+          Get-ChildItem "Install/Win32/install_image" | ForEach-Object { echo "  - $($_.Name)" }
+        } else {
+          echo "❌ install_image directory missing"
+        }
+
+    - name: Upload x64 installer
+      uses: actions/upload-artifact@v4
+      with:
+        name: firebird-odbc-windows-x64-installer
+        path: Install/Win32/install_image/firebird-odbc-windows-x64-installer.exe
+        if-no-files-found: error
+
+  build-linux:
+    runs-on: ${{ matrix.runner }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - arch: x64
+            runner: ubuntu-latest
+          - arch: arm64
+            runner: ubuntu-22.04-arm
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Build Linux ${{ matrix.arch }}
+      run: |
+        # Install dependencies (unixodbc)
+        sudo apt-get update -y
+        sudo apt-get install -y unixodbc unixodbc-dev build-essential
+
+        # Build ODBC driver
+        cd Builds/Gcc.lin
+        cp makefile.linux makefile
+        make
+
+        # Package with documentation
+        mkdir -p firebird-odbc-linux-${{ matrix.arch }}
+        cp Release_*/libOdbcFb.so firebird-odbc-linux-${{ matrix.arch }}/
+        cp ../../Install/IDPLicense.txt firebird-odbc-linux-${{ matrix.arch }}/
+        cp ../../README.md firebird-odbc-linux-${{ matrix.arch }}/
+        tar -czf firebird-odbc-linux-${{ matrix.arch }}.tar.gz firebird-odbc-linux-${{ matrix.arch }}/
+
+    - name: Upload Linux ${{ matrix.arch }} Artifact
+      uses: actions/upload-artifact@v4
+      with:
+        name: firebird-odbc-linux-${{ matrix.arch }}
+        path: Builds/Gcc.lin/firebird-odbc-linux-${{ matrix.arch }}.tar.gz
+
+  create-release:
+    needs: [
+      package-windows-x86,
+      package-windows-x64,
+      installer-windows-x86,
+      installer-windows-x64,
+      build-linux
+    ]
+    runs-on: ubuntu-latest
+    steps:
+    - name: Download all artifacts
+      uses: actions/download-artifact@v4
+      with:
+        merge-multiple: true
+
+    - name: List downloaded artifacts
+      run: |
+        echo "=== Downloaded artifacts ==="
+        find . -type f -name "*.zip" -o -name "*.exe" -o -name "*.tar.gz" | sort
+
+    - name: Create Release
+      uses: softprops/action-gh-release@v2
+      with:
+        files: |
+          firebird-odbc-windows-x86.zip
+          firebird-odbc-windows-x64.zip
+          firebird-odbc-linux-x64.tar.gz
+          firebird-odbc-linux-arm64.tar.gz
+          firebird-odbc-windows-x86-installer.exe
+          firebird-odbc-windows-x64-installer.exe
```

Comment:
The version should be part of the file names of the published artifacts.

Reply:
(I placed it here, but it starts with the file names generated in the other jobs)